### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/TopHandler.hs [hunk 1]
```diff
@@ -74,7 +74,7 @@ toplevelExceptionHandler prog = do
       ]
   where
     -- Let async exceptions rise to the top for the default GHC top-handler.
-    -- This includes things like ctl-c.
+    -- This includes things like CTRL-C.
```
Comment (intricate) [inline cardano-cli/src/Cardano/CLI/TopHandler.hs:77]: Maybe:

```suggestion
    -- This includes things like CTRL-C (SIGINT).
```

File: cardano-node/docs/GenesisData.md [hunk 2]
```diff
@@ -140,7 +140,7 @@ The key `protocolMagic` defines the protocol magic number. When the protocol mag
 
 - `pm` - is the protocol magic number, is included in serialized blocks and headers, and is part of signed data. It is used for differentiating between different clusters.
 
-- `requiresNetworkMagic` - can be `NMMustBeNothing`,`NMMustBeJust`, `RequiresNoMagic`, or `RequiresMagic`. It is essentially a boolean like flag to indicate whether we are on mainnet/staging (`NMMustBeJust`
+- `requiresNetworkMagic` - can be `NMMustBeNothing`,`NMMustBeJust`, `RequiresNoMagic`, or `RequiresMagic`. It is essentially a Boolean like flag to indicate whether we are on mainnet/staging (`NMMustBeJust`
```
Comment (intricate) [inline cardano-node/docs/GenesisData.md:143]: Is lowercase "boolean" really a typo here?

File: cardano-cli/src/Cardano/CLI/TopHandler.hs (hunk 1 above)
Comment (intricate) [reply cardano-cli/src/Cardano/CLI/TopHandler.hs:77]: Although, you are just correcting a typo ("ctl"). Ignore me.

File: cardano-node/docs/GenesisData.md (hunk 2 above)
Comment (newhoggy) [reply cardano-node/docs/GenesisData.md:143]: My spell checker thinks so 😂

File: cardano-api/src/Cardano/Api/TxSubmit/ErrorRender.hs [hunk 3]
```diff
@@ -3,7 +3,7 @@ module Cardano.Api.TxSubmit.ErrorRender
   ( renderApplyMempoolPayloadErr
   ) where
 
--- This file contains error renders. The should hve defined at a lower level, with the error
+-- This file contains error renders. The should have defined at a lower level, with the error
 -- type definitions, but for some reason have not been.
```
[the remark this answers is not in the document]
Comment (newhoggy) [reply cardano-api/src/Cardano/Api/TxSubmit/ErrorRender.hs:7]: Yeah, based on the following sentence, this fix seems right.

File: cardano-node/docs/GenesisData.md (hunk 2 above)
Comment (newhoggy) [reply cardano-node/docs/GenesisData.md:143]: So does Wikipedia.

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**